hack/Dockerfile.buildah

```diff
@@ -0,0 +1,7 @@
+FROM fedora:latest
+RUN dnf -y update; dnf -y clean all
+RUN dnf -y install nginx --setopt install_weak_deps=false; dnf -y clean all
+RUN echo "daemon off;" >> /etc/nginx/nginx.conf
+RUN echo "nginx on Fedora" > /usr/share/nginx/html/index.html
+EXPOSE 80
+CMD [ "/usr/sbin/nginx" ]
```

Comment:
Unless I'm missing a difference, I'd drop this file and just use Dockerfile for both cases.

Reply:
yup, I earlier noticed trouble using just `fedora:latest` for buildah pull, but it seems to be working now. Latest commit removes this `Dockerfile.buildah`